bin/github-deploy.sh

```diff
@@ -158,4 +181,7 @@ git checkout $CURRENTBRANCH
 git branch -D $BRANCH
 git push origin --delete $BRANCH
 
+# regenerate classmap for development
+composer dump-autoload
```

Comment:
All good points - I'll create an todo (in internal "document all the things post") to improve documentation around this. A lot of the complexity here is due to usage of jetpack autoloader and the feature plugin being available in WooCommerce core as a package but _also_ as a regular WordPress plugin. So there is need to ensure that when the feature plugin version is greater than the version bundled with WooCommerce core the feature plugin classes are used instead of the package classes in Woo Core.

A lot of the fixes made in these scripts is to remove the mental overhead of having to remember what needs done for version management but I agree for _maintainance_ of these scripts, it'd be good to document what's happening.